decidim-core/app/cells/decidim/content_blocks/hero_cell.rb

```diff
@@ -18,6 +18,12 @@ def translated_welcome_text
       def background_image
         model.images_container.background_image.big.url
       end
+
+      private
+
+      def cache_hash
+        "decidim/content_blocks/hero/#{Digest::MD5.hexdigest(model.attributes.to_s)}/#{current_organization.cache_version}"
```

Comment:
why don't use `model.updated_at` instead of hashing all the attributes?
shouldn't we take into account also the current locale?

Reply:
Definitely the locale is needed so we have a cached version for each one.

Reply:
Sure, I will be able to work on it on Monday

Reply:
> why don't use `model.updated_at` instead of hashing all the attributes?
> shouldn't we take into account also the current locale?

The model doesn't have an updated_at attribute in this case, nor a cache_version therefore it is needed to hash all attributes.

Reply:
definitely, I just added them

Reply:
> why don't use model.updated_at instead of hashing all the attributes?

I thought exactly the same when reviewing this... @armandfardeau could you add your answer as a comment in the code :pray: ?

Reply:
> > why don't use model.updated_at instead of hashing all the attributes?
> 
> I thought exactly the same when reviewing this... @armandfardeau could you add your answer as a comment in the code 🙏 ?

For sure.